pandas/io/formats/style.py

```diff
@@ -566,7 +567,13 @@ def to_excel(
         inf_rep: str = "inf",
         verbose: bool = True,
         freeze_panes: tuple[int, int] | None = None,
+        storage_options: StorageOptions = None,
     ) -> None:
+        """
+        {storage_options}
```

Comment:
Does this render properly? The docstring is added by the decorator `@doc` above, which is based on to_excel from NDFrame.

Reply:
I think it does. so it's done this way (I see that pattern used on other docs) to overwrite that section from decorator

Reply:
It does not.

![image](https://user-images.githubusercontent.com/45562402/164091468-d4a7a222-2381-4653-9630-bdbec7210a25.png)

[snip]

![image](https://user-images.githubusercontent.com/45562402/164091540-269f964d-43cc-41d3-bba5-46beb1083e92.png)

Reply:
It does not.

![image](https://user-images.githubusercontent.com/45562402/164091643-f84a9820-27d2-4fd7-b507-ce289a0931eb.png)

[snip]

![image](https://user-images.githubusercontent.com/45562402/164091734-7b570a63-7a17-4791-a58e-becb94da98fd.png)